CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Changelog
 
+### [1.6.0.1](https://github.com/stoned/rd-winrm-plugin/releases/tag/1.6.0.1)
```

Comment:
Please update plugin version also in yaml file, and because it is improvement not bugfix better to 1.7.0

Reply:
@stoned would it be possible to update it?